app/javascript/components/cloud-volume-form/index.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useMemo, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+import MiqFormRenderer from '@@ddf';
+import createSchema from './cloud-volume-form.schema';
+import miqRedirectBack from '../../helpers/miq-redirect-back';
+
+const CloudVolumeForm = ({ recordId }) => {
```

Comment:
You need to destructure the `storageManagerId` argument:
```suggestion
const CloudVolumeForm = ({ recordId, storageManagerId }) => {
```